### ISSUE TITLE
Doesn't work with Roomba Combo

### ISSUE THREAD

**Issue by claudiomerli** [karlvr/homebridge-roomba2#125]
### Expected Behavior

Roomba start cleaning when switched on

### Current Behavior

Roomba Combo doesn't start when activated. It beep one time, wifi lights blink, and then nothing. On the homebridge (and homekit as well) side I look it switch on for a moment and then switch off immediately

### Steps to Reproduce (for bugs)

Install plugin, configure with setup documented, try to start device.

### Versions

- **Homebridge**: `v16.16.0`
- **Node**:       `v16.16.0`
- **Plugin**:         v1.3.1

### Configuration

```json
{
    "name": "Roomba",
    "model": "Combo",
    "blid": "****************************",
    "robotpwd": "****************************",
    "ipaddress": "192.168.1.51",
    "dockContactSensor": true,
    "stopBehaviour": "home",
    "accessory": "Roomba2"
}
```

### Homebridge Log / Command Output

```
Starting Roomba
[17/10/2022, 09:45:03] [Roomba] Releasing an unexpected Roomba instance
[17/10/2022, 09:45:12] [Roomba] Starting Roomba
[17/10/2022, 09:45:28] [Roomba] Stopping Roomba
[17/10/2022, 09:45:35] [Roomba] Starting Roomba
[17/10/2022, 09:45:58] [Roomba] Stopping Roomba
[17/10/2022, 09:46:09] [Roomba] Starting Roomba
[17/10/2022, 09:49:15] [Roomba] Stopping Roomba
[17/10/2022, 09:50:30] [Roomba] Timed out after 60001ms trying to connect to Roomba
[17/10/2022, 09:50:30] [Roomba] Releasing an unexpected Roomba instance
[17/10/2022, 09:50:30] [Roomba] Failed to connect to Roomba to refresh state: Connect timed out
[17/10/2022, 09:50:40] [Roomba] Timed out after 60001ms trying to connect to Roomba
[17/10/2022, 09:50:40] [Roomba] Releasing an unexpected Roomba instance
[17/10/2022, 09:50:40] [Roomba] Failed to connect to Roomba to refresh state: Connect timed out
[17/10/2022, 09:50:50] [Roomba] Timed out after 60000ms trying to connect to Roomba
[17/10/2022, 09:50:50] [Roomba] Releasing an unexpected Roomba instance
[17/10/2022, 09:50:50] [Roomba] Failed to connect to Roomba to refresh state: Connect timed out
[17/10/2022, 09:51:00] [Roomba] Timed out after 60000ms trying to connect to Roomba
[17/10/2022, 09:51:00] [Roomba] Releasing an unexpected Roomba instance
[17/10/2022, 09:51:00] [Roomba] Failed to connect to Roomba to refresh state: Connect timed out
[17/10/2022, 09:51:10] [Roomba] Timed out after 60001ms trying to connect to Roomba
[17/10/2022, 09:51:10] [Roomba] Releasing an unexpected Roomba instance
[17/10/2022, 09:51:10] [Roomba] Failed to connect to Roomba to refresh state: Connect timed out
[17/10/2022, 09:51:20] [Roomba] Timed out after 60000ms trying to connect to Roomba
[17/10/2022, 09:51:20] [Roomba] Releasing an unexpected Roomba instance
```

**Comment by Skyman81** [comment]
same error on Roomba 671...

**Comment by Skyman81** [comment]
> Did you upgrade nodejs to v18.x? I was having this exact issue and downgraded to v16.18.1 and it resolved my issues. My guess is this plugin will need to be updated in order to support v18.x (there are a few others where this is causing issues too).
> 
> This can be done by running this command from the terminal on your Homebridge host: _sudo hb-service update-node 16.18.1_

**Comment by timsnaj** [comment]
> > Did you upgrade nodejs to v18.x? I was having this exact issue and downgraded to v16.18.1 and it resolved my issues. My guess is this plugin will need to be updated in order to support v18.x (there are a few others where this is causing issues too).
> > This can be done by running this command from the terminal on your Homebridge host: _sudo hb-service update-node 16.18.1_
> 
> thanks to NateUT99

The downgrade to v16.8.1. did the trick for my new iRobot J7+ Combo. Had to reinstall the plugin after the downgrade. But all is working now. Both my iRobot's show up in HomeKit now. Thanks @Skyman81

**Comment by karlvr** [comment]
Closed as duplicate of #126
